rust/psibase/src/graph_ql.rs

```diff
@@ -374,3 +378,284 @@ pub trait NamedEvent {
 pub trait EventDb {
     fn db() -> DbId;
 }
+
+#[derive(Debug)]
+struct SqlRow {
+    rowid: i32,
```

Comment:
The event id is u64. i32 will eventually overflow for a long running chain.

Reply:
Could have sworn I saw an i32 being used somewhere, because I also thought that seemed small.
Anyway, fixed